applications.yaml

```diff
@@ -1,11 +1,26 @@
 ---
+- name: AhkWindowSpy
```

Comment:
Let's change this to `AutoHotkey` and target `AutoHotkeyU64.exe` because I realised that we also need to set the `try_and_multi_window` option to not leave ghost window tiles behind when a user opens the logs from a running script on the system tray and then closes that window.